tests/monster_test_grpc.js

```diff
@@ -0,0 +1,80 @@
+// Generated GRPC code for FlatBuffers TS *** DO NOT EDIT ***
```

Comment:
why this js file and not others?

Reply:
js files shouldn't be here. Will remove.

Reply:
That's actually a generated file since it's required by grpc to work properly.

Reply:
Oh, ok. Will bring it back.